components/progress-bar/index.jsx

```diff
@@ -101,16 +106,26 @@ class ProgressBar extends React.Component {
 		const labels = assign({}, defaultProps.labels, this.props.labels);
 
 		return (
-			<div id={this.getId()}>
-				{this.getDescription({ labels })}
+			<div
+				id={this.getId()}
+				style={
+					this.props.orientation === 'vertical' ? { height: '200px' } : null
```

Comment:
Instead of hard-coding `200px`, I think we should allow a `style` prop to be passed in so that the consumer can specify a height.  If no height is specified, we could use `100%` as a default.

Example:
```
<ProgressBar
  id="progress-bar-vertical"
  value={60}
  orientation="vertical"
  style={{ height: '200px' }}
/>
```